packages/expo-in-app-purchases/ios/EXInAppPurchases/EXInAppPurchasesModule.h

```diff
@@ -0,0 +1,15 @@
+//  Copyright © 2018 650 Industries. All rights reserved.
+
+#import <UMCore/UMExportedModule.h>
+#import <UMCore/UMModuleRegistryConsumer.h>
+#import <StoreKit/StoreKit.h>
+
+@interface EXInAppPurchasesModule : UMExportedModule <UMModuleRegistryConsumer, SKProductsRequestDelegate, SKPaymentTransactionObserver> {
```

Comment:
Is there a reason to mix properties declaration styles `@interface … { propertiesA } propertiesB @end`?